app/grandchallenge/algorithms/templates/algorithms/job_progress_detail.html

```diff
@@ -2,7 +2,7 @@
 {% load static %}
 
 {% block title %}
-    Algorithm Job Progress - {{ block.super }}
+    Progress - {{ object.pk }} - {{ block.super }}
```

Comment:
Again, not sure if it is clear anymore what progress your looking at if you just print the pk. I think `Progress - {{ object }} - {{ block.super }}` would be better.